test/api_container_create_test.go

```diff
@@ -34,21 +34,18 @@ func (suite *APIContainerCreateSuite) TestCreateOk(c *check.C) {
 		"HostConfig": map[string]interface{}{},
 	}
 
-	path := "/containers/create"
```

Comment:
any reason why delete this variable ? using this variable would be easier to change in the future right ?

Reply:
Yes, what you said is totally right.
Originally, we kept this variable. But when we help our contributors to contribute test codes. We found that "How to help them contribute EASILY and CLEARLY" is the most important thing we should do. 

Removal of this variable is to make the request path and request closer(combined with no lines between them), because this will help user easier to read. 

This is for other's experience of reading code.

While this has not been merged yet. Better solutions are welcomed. And feel free to tell us if you have more ideas and meet any issues when contributing. @Gujarats